api/types.go

```diff
@@ -83,10 +83,10 @@ type GenerateRequest struct {
 type InfillRequest struct {
 
 	// InputPrefix is the text before the infilling position to send to the model.
-  InputPrefix string `json:"input_prefix"`
+	InputPrefix string `json:"input_prefix"`
```

Comment:
why not merge with the previous CL?

Reply:
True, I now squashed the 2 commits, I hope thats what you meant.